project-chip/connectedhomeip
# Added support to the bridge-app for action-lists.

Added the actions to turn on/off lights in a room (endpoint-list).
The instant-action is the supported command to run the actions.
Also, the setup-url attrubute was changed to be active.

#### Problem
Added support to the Linux bridge-app example to tests action-list support for action clusters.

#### Change overview
Commands were added to rename an action-list, remove an action list, and add an action list. Support was also added to run the actions using instant-action. 
The setup-url attribute for the bridge-app actions cluster was enabled, so it can be read.

#### Testing
The changes to the bridge-app were tested by running the chip-bridge-app from a Linux terminal and using chip-tool to examine the action-list for the bridge action cluster.

1. Start bridge app and wait for initialization to complete
2. Run chip-tool pairing
    chip-tool pairing ethernet 12344321 20202021 3840 localhost 5540
3. Get the action-list and observe what lists are available.
    chip-tool bridgedactions read action-list 12344321 1
4. Press m in the bridge-app to rename action 0x1001.
5. Get the action-list to verify the name changed.
    chip-tool bridgedactions read action-list 12344321 1
6. Press n in the bridge-app to remove action 0x1002.
7. Get the action-list to verify the action was removed.
    chip-tool bridgedactions read action-list 12344321 1
8. Press o in the bridge-app to add action 0x1003.
9. Get the action-list to verify the action was added.
    chip-tool bridgedactions read action-list 12344321 1
10. Run action 0x1001 to turn on the lights in Room 1.
    chip-tool bridgedactions instant-action 0x1001 12344321 1 --InvokeID 1
11. Verify state for the lights in Room 1 was changed.
